code-challenges-401/getEdge/README.md

```diff
@@ -0,0 +1,10 @@
+# Find Edges
+
+## Challenge
+- Given a business trip itinerary, and an Alaska Airlines route map, is the trip possible with direct flights? If so, how much will the total trip cost be?
+
+## Approach and Efficiency 
+Big O Space:
+Big O Time:
+
+## API
```

Comment:
Hey Nick, when you have time to come back to this I will regrade for you. Take your time with it.